agent.go

```diff
@@ -1492,6 +1503,7 @@ func realMain() error {
 		deviceWatchers: make(map[string](chan string)),
 		storages:       make(map[string]*sandboxStorage),
 		stopServer:     make(chan struct{}),
+		oomEvents:      make(chan string, 128),
```

Comment:
This channel will aggregate all of the oom event channels from the containers into a single buffered channel. A call to `GetOOMEvents` will read off all of the current events off the channel.